wikimedia/v1/summary_v1.yaml

```diff
@@ -92,9 +94,9 @@ paths:
               uri: /{domain}/sys/action/query
               body:
                 prop: 'extracts|pageimages'
-                redirects: ''
+                redirects: 'true'
```

Comment:
There shouldn't be a need for quoting the value, PHP accepts booleans just fine, AFAIK.